lib/simp/beaker_helpers/ssg.rb

```diff
@@ -71,15 +73,15 @@ class SSG
       },
       'CentOS' => {
         '6' => {
```

Comment:
Why are there hash entries for EL6?

Reply:
@lnemsick-simp Because we used to support it and removing it might break something.